revenuecat_examples/purchase_tester/lib/main.dart

```diff
@@ -36,6 +36,7 @@ Future<void> _configureSDK() async {
   } else {
     configuration = PurchasesConfiguration(StoreConfig.instance.apiKey);
   }
+  configuration.shouldShowInAppMessagesAutomatically = false;
```

Comment:
Bump

Reply:
I think in order for the`showInAppMessages` button to work properly I need to configure the SDK in this way?

Reply:
Hmm that's true, but I think it could be inconvenient to disable the default behavior in our test app. I would leave the default, and maybe a comment on the method indicating that it will only work if the code is changed to disable messages from showing automatically?